src/main/java/edu/cornell/kfs/fp/businessobject/ScheduledAccountingLineUtil.java

```diff
@@ -0,0 +1,114 @@
+package edu.cornell.kfs.fp.businessobject;
+
+import static org.kuali.kfs.sys.document.validation.impl.AccountingDocumentRuleBaseConstants.ERROR_PATH.DOCUMENT_ERROR_PREFIX;
+
+import java.sql.Date;
+import java.util.Calendar;
+import java.util.TreeMap;
+
+import org.apache.commons.lang.StringUtils;
+import org.kuali.kfs.sys.businessobject.AccountingLine;
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+import org.kuali.rice.krad.util.GlobalVariables;
+
+import edu.cornell.kfs.sys.CUKFSConstants;
+import edu.cornell.kfs.sys.CUKFSKeyConstants;
+
+public class ScheduledAccountingLineUtil {
+	public static void copyFrom(ScheduledSourceAccountingLine toLine, AccountingLine other) {
```

Comment:
Does having this method in the util class buy us anything? It would seem more consistent with existing code to just have this logic in ScheduledSourceAccountingLine.copyFrom() itself.